src/client/eppo-precomputed-client.ts

```diff
@@ -43,6 +45,20 @@ export type PrecomputedFlagsRequestParameters = {
   skipInitialPoll?: boolean;
 };
 
+export function convertContextAttributesToSubjectAttributes(
+  contextAttributes: ContextAttributes,
+): Attributes {
+  return {
+    ...(contextAttributes.numericAttributes || {}),
+    ...(contextAttributes.categoricalAttributes || {}),
+  };
+}
+
+interface EppoPrecomputedClientOptions {
+  precomputedFlagStore: IConfigurationStore<PrecomputedFlag>;
+  isObfuscated?: boolean;
```

Comment:
I would also expect `EppoPrecomputedClient` to receive sdk key, subject key, attributes, assignment logger, optional initial configuration, etc in constructor